### PR TITLE
Exposed amqplib message properties to the consumer. 

### PR DESCRIPTION
**Pull request by aviz**
The consumer should have access to the message properties and specifically to be able to read headers.

### CODE REVIEW

File: test/producer-consumer-spec.js [hunk 1]
```diff
@@ -47,6 +47,29 @@ describe('producer/consumer', function () {
       });
     });
 
+
+    it('should receive message headers', () => {
+      const header =  {header1: "Header1", header2: "Header2"};
```
Comment (mrister) [inline test/producer-consumer-spec.js]: minor: consider naming this `headers`, it has 2 of them and you could then use a shorthand notation on line 55

**Review by mrister — APPROVED** [review]

**Review by jkernech — APPROVED** [review]
Aside fom what was already mentioned by @mrister, LGTM

**Review by ykhatal — APPROVED** [review]
Thanks for the contribution.
Aside from noted, LGTM

File: test/producer-consumer-spec.js (hunk 1 above)
Comment (Spring3) [reply test/producer-consumer-spec.js]: In fact, if you take a look at the message structure:

Comment (Spring3) [reply test/producer-consumer-spec.js]: And judging by the data inside these `properties`, there are not exactly just headers, also some options passed to the consumer, so I would not rename it

**Review by Spring3 — APPROVED** [review]

File: test/producer-consumer-spec.js (hunk 1 above)
Comment (aviz) [reply test/producer-consumer-spec.js]: renamed.

**Review by ofkindness — APPROVED** [review]

**Comment by aviz** [comment]
When can you merge it and release a new version ? I rather use the main repo than a fork.

**Comment by mrister** [comment]
@dial-once/developers can anyone do a release to not leave our nice contributor waiting?

**Comment by Spring3** [comment]
@aviz thanks again for your input

**Pull request merged**